### PR TITLE
Support 'web app' payload for InlineKeyboardButton 

### PR DESCRIPTION
**Pull request by nyavro**
According to Telegram Bot API 6 0

### CODE REVIEW

File: telegram/src/main/kotlin/com/github/kotlintelegrambot/network/serialization/adapter/InlineKeyboardButtonAdapter.kt [hunk 1]
```diff
@@ -39,6 +40,7 @@ internal class InlineKeyboardButtonAdapter : JsonSerializer<InlineKeyboardButton
         is SwitchInlineQueryCurrentChat -> context.serialize(src, SwitchInlineQueryCurrentChat::class.java)
         is CallbackGameButtonType -> context.serialize(src, CallbackGameButtonType::class.java)
         is Pay -> context.serialize(src, Pay::class.java)
+        is WebApp -> context.serialize(src, WebApp::class.java)
```
Comment (vjgarciag96) [inline telegram/src/main/kotlin/com/github/kotlintelegrambot/network/serialization/adapter/InlineKeyboardButtonAdapter.kt:43]: Is it worth adding a test case in `InlineKeyboardButtonAdapterTest` for this?

**Review by vjgarciag96 — COMMENTED** [review]
Nice! Thanks for the contribution, happy to merge this once we have a small test for the serialization in place!.

Also, a couple of other things to mention:

* I see in the doc this can also be part of `KeyboardButton`, maybe it's worth moving `WebAppInfo` to a separated file to reuse it, but that can be done separately when support for `KeyboardButton` is added.
* We haven't added support for deserializing `WebApp` fields in `InlineKeyboardButtonAdapter`. This can also be done separately.

I can create issues for these two points, they don't neccesarily need to be implemented in this PR.

File: telegram/src/main/kotlin/com/github/kotlintelegrambot/network/serialization/adapter/InlineKeyboardButtonAdapter.kt (hunk 1 above)
Comment (nyavro) [reply telegram/src/main/kotlin/com/github/kotlintelegrambot/network/serialization/adapter/InlineKeyboardButtonAdapter.kt:45]: - Added `InlineKeyboardButtonAdapterTest`
- Fixed `KeyboardButton`
- Added deserialization of WebApp

Please review and check if I messed something.
Regards

**Review by vjgarciag96 — APPROVED** [review]
Looks great! Thank you so much for the contribution @nyavro

**Comment by nyavro** [comment]
@vjgarciag96 When do you plan to bump the library version in JitPack?

**Comment by vjgarciag96** [comment]
I can do a minor release now if you want to use this change, let me prepare it @nyavro

**Comment by vjgarciag96** [comment]
Published now https://github.com/kotlin-telegram-bot/kotlin-telegram-bot/releases/tag/6.0.7. You can find it in https://jitpack.io/#kotlin-telegram-bot/kotlin-telegram-bot/6.0.7. Let me know if you find any issues @nyavro

**Comment by nyavro** [comment]
@vjgarciag96 Great! Thank you!

**Pull request merged**